src/sonic-host-services/scripts/sonic-host-server

```diff
@@ -13,9 +13,16 @@ import dbus.mainloop.glib
 
 from gi.repository import GObject
 
-def register_modules():
+def find_module_path():
+    """Find path for host_moduels"""
+    for path in sys.path:
+        mod_path = path + '/host_modules'
+        if os.path.exists(mod_path):
+            return mod_path
+    return None
+
+def register_modules(mod_path):
     """Register all host modules"""
-    mod_path = '/usr/local/lib/python3.7/dist-packages/host_modules'
```

Comment:
>mod_path [](http://example.com/codeflow?start=4&length=8)

Did you try [https://stackoverflow.com/a/248862/2514803](https://stackoverflow.com/a/248862/2514803)?

Reply:
sonic-host-services does not have any module to import, can I import other module instead? For example, dbus